server/audit_math/macro.py

```diff
@@ -159,14 +158,7 @@ def compute_U(
     """
     U = Decimal(0.0)
     for batch in reported_results:
-        if batch in sample_results:
-            error = compute_error(
-                reported_results[batch], sample_results[batch], contest
-            )
```

Comment:
It looks like L57 of this file is now not covered by any test, so the test coverage check is failing. My guess is that it's a result of removing this line (or maybe removing a test). In any case, the check seems unnecessary to me - we should never be computing the error for a batch without having its reported results, so I think it's probably safe to just remove that line.

Reply:
Agreed.